example/l2fwd_simple/l2fwd_simple_run.sh

```diff
@@ -6,32 +6,34 @@
 # SPDX-License-Identifier:     BSD-3-Clause
 #
 
-PCAP_IN=`find . ${TEST_DIR} $(dirname $0) -name udp64.pcap -print -quit`
-echo "using PCAP_IN = ${PCAP_IN}"
+if  [ -f ./interface_env ]; then
+	. ./interface_env
+else
+        echo "BUG: unable to find interface_env!"
+        echo "interface_env has to be in current directory"
+        exit 1
+fi
+
+setup_interfaces
 
-./odp_l2fwd_simple${EXEEXT} pcap:in=${PCAP_IN} pcap:out=pcapout.pcap \
-	02:00:00:00:00:01 02:00:00:00:00:02 -t 2
+./odp_l2fwd_simple${EXEEXT} $IF0 $IF1 02:00:00:00:00:01 02:00:00:00:00:02 -t 2
 STATUS=$?
 
 if [ "$STATUS" -ne 0 ]; then
   echo "Error: status was: $STATUS, expected 0"
   exit 1
 fi
 
-if [ `stat -c %s pcapout.pcap` -ne `stat -c %s  ${PCAP_IN}` ]; then
-  echo "File sizes disagree"
-  exit 1
-fi
-
-rm -f pcapout.pcap
+validate_result
 
-./odp_l2fwd_simple${EXEEXT} null:0 null:1 \
-	02:00:00:00:00:01 02:00:00:00:00:02 -t 2
+./odp_l2fwd_simple${EXEEXT} $IF2 $IF3 02:00:00:00:00:01 02:00:00:00:00:02 -t 2
```

Comment:
This null interface test doesn't make much sense, so it could be removed.

Reply:
Ack.